src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -67,23 +86,89 @@ const getInstallation = async (subscription: Subscription, log: Logger, gitHubAp
 	}
 };
 
-interface FailedConnection {
-	id: number;
-	deleted: boolean;
-	orgName?: string;
-}
+const getConnectionsAndInstallations = async (subscriptions: Subscription[], req: Request, id: number): Promise<ConnectionsAndInstallations> => {
+	const installations = await getInstallations(subscriptions, req.log, id);
+
+	const failedConnections: FailedConnection[] = await Promise.all(
+		installations.rejected.map(async (installation) => {
+			const sub = subscriptions.find((sub: Subscription) => installation.id === sub.gitHubInstallationId);
+			const repo = sub && await RepoSyncState.findOneFromSubscription(sub);
+			return {
+				id: installation.id,
+				deleted: installation.deleted,
+				orgName: repo?.repoOwner
+			};
+		}));
+
+	const successfulConnections: SuccessfulConnection[] = installations.fulfilled
+		.map((installation) => ({
+			...installation,
+			isGlobalInstall: installation.repository_selection === "all"
```

Comment:
I'm still not sure what a 'global install' is

Reply:
This part of the code was just moved and not changed.